src/emailpro/disclaimer/update/emailpro-disclaimer-update.html

```diff
@@ -18,7 +18,7 @@
                         <label for="variableSelect" class="oui-field__label oui-label"
                                data-translate="emailpro_ACTION_disclaimer_insert"></label>
                         <div class="oui-field__content">
-                            <div class="oui-select">
+                            <div class="oui-select mb-0 mr-3">
```

Comment:
Please refer to my previous comment.

Reply:
This is done. Thanks a lot for the suggestion :)